examples/workspaces_variables/README.md

```diff
@@ -34,9 +34,9 @@ Have the following information handy for the next steps:
 
 The `manager` workspace requires certain Terraform variables to be set on the [Variables page in the UI](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/variables#managing-variables-in-the-ui). All variables should be "Terraform Variables". There are no "Environment Variables" required.
 
-* `tf_hostname` - The Terraform Cloud or Enterprise hostname. Defaults to app.terraform.io
-* `tf_api_token` - The TFC/TFE user token that was generated. Mark this as sensitive.
-* `tf_organization` - The TFC/TFE organization name where the managed workspaces should exist. This will most commonly be the organization that was created or joined, where the `manager` workspace also exists. The token supplied to perform the operations should have sufficient access to the organization.
+* `tf_hostname` - The HCP Terraform or Enterprise hostname. Defaults to app.terraform.io
+* `tf_api_token` - The HCP Terraform and Terraform Enterprise user token that was generated. Mark this as sensitive.
+* `tf_organization` - The HCP Terraform and Terraform Enterprise organization name where the managed workspaces should exist. This will most commonly be the organization that was created or joined, where the `manager` workspace also exists. The token supplied to perform the operations should have sufficient access to the organization.
```

Comment:
```suggestion
* `tf_organization` - The HCP Terraform or Terraform Enterprise organization name where the managed workspaces should exist. This will most commonly be the organization that was created or joined, where the `manager` workspace also exists. The token supplied to perform the operations should have sufficient access to the organization.
```